client/modules/IDE/reducers/files.js

```diff
@@ -116,6 +116,26 @@ function sortedChildrenId(state, children) {
   return childrenArray.map(child => child.id);
 }
 
+function udpateParent(state, action) {
```

Comment:
Thanks for doing the refactor! One small thing—I think you meant `"update"` here rather than `"udpate"`.

Reply:
oh sorry wait just updating

Reply:
no worries!!

Reply:
done!!